docs/develop/03_workshop/04_attester/02_did.md

```diff
@@ -11,64 +11,98 @@ import TabItem from '@theme/TabItem';
 import GenerateKeypairs from '!!raw-loader!@site/code_examples/sdk_examples/src/workshop/attester/generateKeypairs.ts';
 import GenerateDid from '!!raw-loader!@site/code_examples/sdk_examples/src/workshop/attester/generateDid.ts';
 
-Now it's time to generate a DID using the previously created account for the <span className="label-role attester">Attester</span>.
+The next step is to generate a KILT decentralized identifier (DID) using the account you created for the <span className="label-role attester">Attester</span> in [the previous step](./01_account.md).
 
-A DID may represent any entity, which could be a person, an organization or a machine.
+<!-- TODO: Also factor into quickstart -->
```

Comment:
What do you want to add here?

Reply:
@Dudleyneedham An internal note to self to check that some of this good intro content was actually in the quickstart

Reply:
Will move elsewhere

Reply:
```suggestion
```